aidial_assistant/application/assistant_application.py

```diff
@@ -83,6 +95,87 @@ def _validate_messages(messages: list[Message]) -> None:
         )
 
 
+def _validate_request(request: Request) -> None:
+    _validate_messages(request.messages)
+    _validate_addons(request.addons)
+
+
+def _construct_function(name: str, description: str) -> Tool:
+    return construct_function(
+        name,
+        description,
+        {
+            "query": {
+                "type": "string",
+                "description": "A task written in natural language",
+            }
+        },
+        ["query"],
+    )
+
+
+def _convert_commands_to_tools(
```

Comment:
Let's keep serializers and deserializers together.
Move the method to `tools_chain.py`, where the messages were serialized to the state in the first place.

Reply:
Moved.